client/src/components/Overview/ImageExpandModal.jsx

```diff
@@ -10,6 +10,32 @@ export default function ImageExpandModal({
   maxIndex,
 }) {
   const { showImageModal, setShowImageModal } = useContext(ProductContext);
+  const imageRef = useRef();
+  const [mousePosition, setMousePosition] = useState({ left: 0, top: 0 });
+  const [dimensions, setDimensions] = useState({ width: 0, height: 0 });
+
+  function handleMouseMove(event) {
+    console.log(event.target);
+    console.log(imageRef)
+    setMousePosition({ left: event.pageX, top: event.pageY });
```

Comment:
store mousePosition relative to div? (subtract off div's position)

Reply:
shoulda read this comment.. :/